_episodes/01-getting-started-with-conda.md

```diff
@@ -37,13 +37,13 @@ When working with a programming language, such as Python, that can do almost _an
 
 ### Dependencies
 
-A bit further into your programming career you may notice/have noticed that many packages do not just do everything on their own. Instead, they _depend_ on other packages for their functionality. For example, the [Scipy package](https://www.scipy.org/about.html) is used for numerical routines. To not reinvent the wheel, the package makes use of other packages, such as numpy (numerical python) and matplotlib (plotting) and many more. So we say that numpy and matplotlib are *dependencies* of Scipy. 
+A bit further into your programming career you may notice/have noticed that many packages do not just do everything on their own. Instead, they _depend_ on other packages for their functionality. For example, the [`Scipy` package](https://www.scipy.org/about.html) is used for numerical routines. To not reinvent the wheel, the package makes use of other packages, such as `numpy` (numerical python) and `matplotlib` (plotting) and many more. So we say that `numpy` and `matplotlib` are *dependencies* of `Scipy`. 
```

Comment:
I realise its inherited but perhaps lower-case `scipy` to be consistent with `numpy` and `matplotlib`.

Reply:
Good point.